cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -204,6 +209,15 @@ func run(ctx context.Context, target, assetType, optJSON string, state checkstat
 		}
 		trivyArgs = append(trivyArgs, []string{"--scanners", sc}...)
 
+		if opt.ImageChecks.Secret {
+			if !opt.DisableCustomSecretConfig {
```

Comment:
Why do we use negative logic here? Zero value defaults?

Reply:
That's a good question. I just copied it from [the previous existing code](https://github.com/adevinta/vulcan-checks/blob/2c1ecbfdfdfcdadd5d2fbd3ce1c5eff50c158936/cmd/vulcan-trivy/vulcan-trivy.go#L315-L318) for `GitRepository`.